pkg/api/v1alpha1/edgeconnect/edgeconnect_types.go

```diff
@@ -91,6 +99,26 @@ type ImageRefSpec struct {
 	Tag string `json:"tag,omitempty"`
 }
 
+type ProxySpec struct {
+	// Proxy scheme.
```

Comment:
```suggestion
	// Proxy scheme (`http` or `https`). 
```

The actual format has probably not been decided yet, but it would be nice to document what is exactly expected.
Would it be possible to have a "special" type alias, that only allows `http` or `https` (does it already exist in some standard lib)? 🤔